# no-literal-call

ãŠã‹ã—ã„æ§‹æ–‡ã‚’ãƒã‚§ãƒƒã‚¯ã™ã‚‹

# Good ðŸ‘

```javascript
foo();

obj.foo();

(function() {})();

// es6
(() => 0)();
```

# BAD ðŸ‘Ž

```javascript

true();

false();

null();

100();

\"hello\"();

/abc/();

[1,2,3]();

({foo: 0})();

`hello`();

(class A {})();
```
