# no-var

var å®£è¨€ã‚’è­¦å‘Šã™ã‚‹

# Good ðŸ‘

```javascript
const foo = "foo";

let foo;

let foo = "foo";

const foo = () => {};

let foo = () => {};
```

# BAD ðŸ‘Ž

```javascript
var foo;

var foo = "foo";

var foo = () => {};
```
